googleapiclient/discovery.py

```diff
@@ -698,21 +698,15 @@ def _cast(value, schema_type):
       A string representation of 'value' based on the schema_type.
     """
     if schema_type == "string":
-        if type(value) == type("") or type(value) == type(""):
```

Comment:
This looks weird, but also potentially unsafe to remove.

Reply:
This is indeed weird, but is also safe.

Look at the last time it was changed (you will have to expand `discover.py` and scroll to line 701): https://github.com/googleapis/google-api-python-client/commit/3bbefc1352bcb2e302f7736643c9363799d5f5df#diff-b039ba2bb64e79cf4c77e4fe5a4b21a826172e8b7ae9be872fd2d46047c05cbbL695-R701

Originally, it was 

```python3
if type(value) == type("") or type(value) == type(u""):
```

But was changed automatically to be

```python3
if type(value) == type("") or type(value) == type(""):
```

Because `type("") == type(u"") == "str"`, this is certainly safe.